iroh/src/node.rs

```diff
@@ -377,6 +375,9 @@ impl<D: iroh_blobs::store::Store> NodeInner<D> {
 
         // Abort remaining tasks.
         join_set.shutdown().await;
+
+        // Abort remaining local tasks.
+        // local_pool.shutdown().await;
```

Comment:
why is this commented out?

Reply:
It is an experiment if this will fix the CI issues. Next will be leak 🙄.

This will not remain in the PR when we merge it...